### DISCUSSION TITLE
Configurable block substitution during cavegen

### DISCUSSION THREAD

**Discussion by Biviho** [yungnickyoung/Better-Caves#79] [Suggestions]
hi, i wanted to ask if it's possible to expand the "gravel to andesite" and "sand to sandstone" on cave generation to use custom blocks on configurable blocks, and, if possible, to also define specific block for when the changed block is exposet to the sky or not.

for example, with tis i should be able to change dirt to have stone underneath, and if the changed dirt is exposed to the sky, to change it to overgrown stone from biomes o plenty by adding something like:

minecraft:dirt, minecraft:stone, biomesoplenty:grass:1

i know this is a pretty specific request, and i'm totally ok if you will deny it.

**Comment by yungnickyoung** [comment]
This probably is a little too specific for me to add it, but I'll keep it open just in case.
